src/components/NavigationOutbox.vue

```diff
@@ -60,11 +76,55 @@ export default {
 			}
 		},
 	},
+	watch: {
+		attempts() {
+			if (this.attempts <= RETRY_COUNT && this.retry) {
+				setTimeout(() => {
+					this.retrySendMessages()
+				}, RETRY_TIMEOUT)
+
+			} else {
+
+				this.retry = false
+				this.attempts = 0
+			}
+		},
+	},
+	created() {
```

Comment:
this means with every page load we retry sending those messages. I'm not sure if this is how we want it.

shouldn't each  message have some kind of retry counter so it's only retried a few times? with the first iteration of the outbox feature we continuously retried sending failed messages. but that caused more trouble than expected. so this was deliberately disabled via https://github.com/nextcloud/mail/pull/6602.

also see https://github.com/nextcloud/mail/issues/6540. there are some side effects with blindly resending ever pending message in the outbox.

Reply:
Let's try with a counter :+1:

Reply:
**Hello everyone!** 
Is it better to turn on the counter when the page loads, or to attract something like a permanent storage (localstorage)? Although probably the latter has more chances for bugs?

Reply:
I'd say the counter should be in the backend